lib/util/memcache.ex

```diff
@@ -20,6 +60,11 @@ defmodule Antikythera.Memcache do
   alias AntikytheraCore.Ets.Memcache
   alias Antikythera.Memcache.{Key, Value, NormalizedFloat}
 
+  @doc """
+  Read the value associated with the `key` from memcache.
+
+  Please note that records in memcache could be evicted anytime so the error handling must be needed.
+  """
   defun read(key :: term, epool_id :: v[EPoolId.t]) :: R.t(term, :not_found) do
```

Comment:
The typespec can be improved here:
- `key :: Key.t`
- return value `:: R.t(Value.t, :not_found)`

Reply:
I forgot to change them..

Fixed in e8adcb50f657f23724421e7b0383d11d95fe9bb0